sormas-backend/src/main/java/de/symeda/sormas/backend/contact/ContactService.java

```diff
@@ -634,22 +635,11 @@ public List<DashboardContactDto> getContactsForDashboard(Region region, District
 
```

Comment:
Could you please add a small unit test for ContactService.getContactsForDashboard with zero data and with one contact and visit to check that the queries are working?